### PR TITLE
Fix OCP-12400

### PR DESCRIPTION
**Pull request by xiuwang**
@openshift/devexp-qe please help review, thx

### CODE REVIEW

**Comment by wzheng1** [comment]
Thanks for the implementation! LGTM

File: features/step_definitions/registry.rb [hunk 1]
```diff
@@ -71,9 +71,9 @@
   end
   begin
     step %Q/I switch to cluster admin pseudo user/
-    step %Q/I use the "default" project/
+    step %Q/I use the "openshift-image-registry" project/
```
Comment (liangxia) [inline features/step_definitions/registry.rb]: The project name change works for all 4.x versions ?

Comment (xiuwang) [reply features/step_definitions/registry.rb]: For now, it's yes

Comment (liangxia) [reply features/step_definitions/registry.rb]: ack

**Pull request merged**